src/app/beer_garden/scheduler.py

```diff
@@ -25,7 +25,6 @@
 from pathtools.patterns import match_any_paths
```

Comment:
There are no actual changes in this file, it should not be included with the PR. If they are required for formatting, then create a separate PR to update our baseline for formatting.